.github/workflows/build-syft-image.yml

```diff
@@ -0,0 +1,51 @@
+name: Build syft image
```

Comment:
We want to build the image in RHTAP, not in GH workflows right? Do we have/can you file a story for that?

Reply:
We discussed this on monday with Mykola and Shoubik, and for now this is the easier option. Other teams currently use github workflows for building as well. Eventually we want to migrate to RHTAP itself though.

Reply:
Huh, ok. I see we like our build pipeline :smile: